components/centraldashboard/public/components/namespace-selector.js

```diff
@@ -153,6 +154,24 @@ export class NamespaceSelector extends PolymerElement {
         return selected;
     }
 
+    /**
+     * Check if role is owner
+     */
+    validate() {
+        const {namespaces} = this;
+        if (!namespaces) return;
+        const nsSet = new Set(namespaces.map((i) => i.namespace));
+        if (nsSet.has(this.selected)) return;
+
+        const owned = namespaces.find((n) => n.role == 'owner');
+        // eslint-disable-next-line
+        console.log({own: (owned||[]).namespace, namespaces, ns: this.selected, yeah: namespaces.includes('kubeflow')});
```

Comment:
I don't think there are any other `console.log` statements for non-error conditions so I'd suggest removing ithis.

Reply:
Good catch, I forgot to remove this line. Thanks!

Reply:
Good catch, I forgot to remove this before submitting the PR. Thanks!